src/Tools/Test Framework/Test Runner/src/CommandLineTestTool.Page.al

```diff
@@ -96,6 +96,14 @@ page 130455 "Command Line Test Tool"
                     if Rec.FindFirst() then;
                 end;
             }
+            field(RequiredTestIsolation; RequiredTestIsolation)
+            {
+                ApplicationArea = All;
+                Caption = 'Required Test Isolation';
+                ToolTip = 'Specifies the Required Test Isolation';
```

Comment:
Could you please create a user-friendlier tooltip?
Something like (I don't know how it works, please fix): "Specifies the test isolation level in which the test codeunit will be run."

Shouldn't `RequiredTestIsolation` rather be an enum than an integer? That way, you could also get rid of the `MinValue` and `BlankZero` properties.